app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,5 @@
 @import "bootstrap";
 @charset "UTF-8";
+@import "form";
+@import "quiz";
```

Comment:
The style.scss file had too much and I couldn't follow it so I pulled the styling that I have done and just created new style sheets so I can follow along with what I've been doing with FE

Reply:
That makes sense, the bootstrap we pulled in was pretty big to start